### PR TITLE
Migrated Bus demo to 1.11.

### PR DESCRIPTION
**Pull request by joerg84**

### CODE REVIEW

**Comment by joerg84** [comment]
Also migrated docker images, last artifact to be migrated is the hosted spark job jar file.

File: fastdata-iot/1.11/README.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+
+### Cassandra
+
+Install DC/OS Apache Cassandra package:
+
+```bash
+dcos package install cassandra
+```
+
+Once it sucessfully deploye, check the dcos cassandra subcommand:
```
Comment (pleia2) [inline fastdata-iot/1.11/README.md]: Missing the "d" at the end of "deployed"

File: fastdata-iot/1.11/README.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+
+```bash
+dcos cassandra endpoints native-client
+```
+
+The setup of the required cassandra schema is done via an [Jobs](https://docs.mesosphere.com/1.11/deploying-jobs/) job.
```
Comment (pleia2) [inline fastdata-iot/1.11/README.md]: should be "via a" not "via an"

File: fastdata-iot/1.11/README.md [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+  "cpus": 0.1,
+  "mem": 2048,
+  "disk": 0,
+  "instances": 1,
+  "container": {
+    "type": "MESOS",
+    "volumes": [],
+    "docker": {
+      "image": "mesosphere/akka-ingest:0.2.1-SNAPSHOT",
+      "network": "HOST",
```
Comment (pleia2) [inline fastdata-iot/1.11/README.md]: This JSON isn't working for me, I was getting an error: container.docker.network: must be empty

I removed this ""network": "HOST"," line and I was able to proceed.

File: fastdata-iot/1.11/configuration/akka-ingest.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  "cpus": 0.1,
+  "mem": 2048,
+  "disk": 0,
+  "instances": 1,
+  "container": {
+    "type": "MESOS",
+    "volumes": [],
+    "docker": {
+      "image": "mesosphere/akka-ingest:0.2.1-SNAPSHOT",
+      "network": "HOST",
```
Comment (pleia2) [inline fastdata-iot/1.11/configuration/akka-ingest.json]: Remove this network line.

File: fastdata-iot/1.11/configuration/dashboard.json [hunk 5]
```diff
@@ -0,0 +1,37 @@
+{
+  "id": "/bus-demo/dashboard",
+  "container": {
+    "type": "MESOS",
+    "docker": {
+      "image": "mesosphere/akka-server:0.2.1-SNAPSHOT",
+      "network": "HOST",
```
Comment (pleia2) [inline fastdata-iot/1.11/configuration/dashboard.json]: Remove this network line.

File: fastdata-iot/1.11/README.md [hunk 6]
```diff
@@ -0,0 +1,235 @@
...
+Either install it via the DC/OS UI by creating a new marathon app:
+
+```json
+{
+  "id": "/bus-demo/dashboard",
+  "container": {
+    "type": "MESOS",
+    "docker": {
+      "image": "mesosphere/akka-server:0.2.1-SNAPSHOT",
+      "network": "HOST",
```
Comment (pleia2) [inline fastdata-iot/1.11/README.md]: Remove this network line.

**Pull request merged**